bemuse/src/music-previewer/index.js

```diff
@@ -24,6 +24,16 @@ export function preview(url) {
   return getInstance().preview(url)
 }
 
+function playAudio(element) {
+  return new Promise((resolve, reject) => {
+    try {
+      resolve(element.play())
+    } catch (e) {
+      reject(e)
+    }
+  })
```

Comment:
The Promise constructor already catches errors that is thrown synchronously in the constructor callback, and makes the promise automatically reject. So, it is possible to do this:

```suggestion
  return new Promise((resolve) => {
    resolve(element.play())
  })
```